src/app/electra/reducer.ts

```diff
@@ -27,6 +27,23 @@ export default function electraReducer(state = initialState, action: any) {
         ...state
       }
     }
+    case ActionNames.GET_GLOBAL_BALANCE_SUCCESS: {
+      return {
+        ...state,
+        electraJs: {
+          ...state.electraJs,
+          wallet: {
+            ...state.electraJs.wallet,
```

Comment:
Keep the reducers for the overview "feature" in the overviewReducer. This will let us keep the electraJs object clean while having a specific reducer to specific states.